.github/workflows/samples-tools/pkg/utils/map.go

```diff
@@ -0,0 +1,23 @@
+package utils
+
+import "sync"
+
+func Map[a, b any](parallel bool, items []a, fn func(a) b) []b {
```

Comment:
question: it looks like `Map` is only used in one place, `runAll()`. Is that correct?

Reply:
Also be sure to provide function-level Godoc comments.

Reply:
Added Godoc.

Yes, it's only used in `runAll`, but it seems like a generic enough function to be in utils. But I can move it to that file, and if we need it at some point for other tools we can always move it to utils.